maapcopcguide (2).ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<ol><li>Cloud-Optimized Point Cloud (COPC) format on the MAAP ADE -&gt; Cloud-Optimized Point Cloud (COPC) format in the MAAP ADE</li><li>Within your Jupyter Notebook, start by importing the maap package. Then invoke the MAAP constructor, setting the maap_host argument to 'api.ops.maap-project.org'.. Maybe have <code>maap_host</code> and <code>api.ops.maap-project.org</code> as inline code blocks.   </li><li>Replace <code>api.ops.maap-project.org</code> by <code>api.maap-project.org</code>   <span class="ql-cursor">﻿</span></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>